doc/development_process/project_roles.rst

```diff
@@ -210,13 +240,16 @@ Roles / Permissions
     ================ =================== =========== ================ =========== =========== ============
 
 
+.. _maintainers_file:
+
 MAINTAINERS File
 ################
 
 Generic guidelines for deciding and filling in the Maintainers' list
 
-* The MAINTAINERS file shall replace the CODEOWNERS file and will be used
-  for both setting assignees and reviewers.
+* The :zephyr_file:`MAINTAINERS.yaml` file shall replace the
+  :zephyr_file:`CODEOWNERS` file and will be used for both setting assignees and
+  reviewers.
```

Comment:
@nashif do we really want to merge this in this PR? It seems premature, since #38725 is still open.

Reply:
Note that I'm only adding file links to the existing text here.

Reply:
resolved offline